### ISSUE TITLE
extensions: UI cannot be navigated using tabs

### ISSUE THREAD

**Issue by pelikhan** [microsoft/pxt-microbit#4571]
I was trying to tab through the filters and extensions, but there is no visual feedback where the cursor focus is.
